internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+TransmitterAddress = '0xa0788FC17B1dEe36f057c42B6F373A34B014687e' # Example
+TraceLogging = false # Default
+
+# P2P supports multiple networking stack versions. You may configure `[P2P.V1]`, `[P2P.V2]`, or both to run simultaneously.
+# If both are configured, then for each link with another peer, V2 networking will be preferred. If V2 does not work, the link will
+# automatically fall back to V1. If V2 starts working again later, it will automatically be preferred again. This is useful
+# for migrating networks without downtime. Note that the two networking stacks _must not_ be configured to bind to the same IP/port.
+#
+# All nodes in the OCR network should share the same networking stack.
+[P2P]
+IncomingMessageBufferSize = 10 # Default
+OutgoingMessageBufferSize = 10 # Default
+
+[P2P.V1]
+# Should be set as the externally reachable IP address of the Chainlink node.
+AnnounceIP = '1.2.3.4' # Example
+# Should be set as the externally reachable port of the Chainlink node.
+AnnouncePort = 1337 # Example
+BootstrapCheckInterval = '20s' # Default
+# Default set of bootstrap peers.
+DefaultBootstrapPeers = ['/dns4/example.com/tcp/1337/p2p/12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U', '/ip4/1.2.3.4/tcp/9999/p2p/12D3KooWLZ9uTC3MrvKfDpGju6RAQubiMDL7CuJcAgDRTYP7fh7R'] # Example
+DHTAnnouncementCounterUserPrefix = 0 # Default
+DHTLookupInterval = 10 # Default
+# The default IP address to bind to.
+ListenIP = '0.0.0.0' # Default
+# The port to listen on. If left blank, the node randomly selects a different port each time it boots. It is highly recommended to set this to a static value to avoid network instability.
+ListenPort = 1337 # Example
+NewStreamTimeout = '10s' # Default
+# The default peer ID to use for OCR jobs. If unspecified, uses the first available peer ID.
+PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw' # Example
+PeerstoreWriteInterval = '5m' # Default
+
+[P2P.V2]
+# The addresses the peer will advertise on the network in host:port form as accepted by net.Dial. The addresses should be reachable by peers of interest.
+AnnounceAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
+# The default bootstrapper peers for libocr's v2 networking stack.
+DefaultBootstrappers = ['12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U@1.2.3.4:9999', '12D3KooWM55u5Swtpw9r8aFLQHEtw7HR4t44GdNs654ej5gRs2Dh@example.com:1234'] # Example
+DeltaDial = '15s' # Default
+DeltaReconcile = '1m' # Default
+# The addresses the peer will listen to on the network in `host:port` form as accepted by `net.Listen()`, but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
+ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
+
+[Keeper]
+DefaultTransactionQueueDepth = 1 # Default
```

Comment:
** advanced/dev only **

DefaultTransactionQueueDepth controls the queue size for DropOldestStrategy in Keeper. Set to 0 to use SendEvery strategy instead.